letta/client/client.py

```diff
@@ -428,7 +428,7 @@ def __init__(
         super().__init__(debug=debug)
         self.base_url = base_url
         self.api_prefix = api_prefix
-        self.headers = {"accept": "application/json", "authorization": f"Bearer {token}"}
+        self.headers = {"accept": "application/json", "X-BARE-PASSWORD": f"password {token}"}
```

Comment:
So we actually need the `token` field for cloud users - would you be able to instead make `password` an additional field, where *either* password or token will be used?